src/metabase/server/middleware/security.clj

```diff
@@ -153,7 +161,8 @@
     ;; Prevent Flash / PDF files from including content from site.
     "X-Permitted-Cross-Domain-Policies" "none"
     ;; Tell browser not to use MIME sniffing to guess types of files -- protect against MIME type confusion attacks
-    "X-Content-Type-Options"            "nosniff"}))
+    "X-Content-Type-Options"            "nosniff"}
+    (access-control-headers)))
```

Comment:
IDK, maybe it makes sense to move it up where content-security-policy is? So that non-literal parts of headers are bundled together.

I also think that moving condition (the `(when (embedding-app-origin) ...` part) here inline is better than hiding it inside the function itself, it's more inline with how the rest of the function is written.

Reply:
I know it's different headers, I'm just talking about their position in that `merge` call. :)